cmd/nerdctl/build_test.go

```diff
@@ -287,3 +287,58 @@ CMD ["echo", "nerdctl-build-test-string"]
 	base.Cmd("run", "--rm", imgWithNoTag).AssertOutExactly("nerdctl-build-test-string\n")
 	base.Cmd("run", "--rm", imgWithCustomTag).AssertOutExactly("nerdctl-build-test-string\n")
 }
+
+func TestBuildWithContainerfile(t *testing.T) {
```

Comment:
Why is test passing `[test-integration-docker-compatibility](https://github.com/containerd/nerdctl/runs/7312918381?check_suite_focus=true#logs)` ?
I don't think Docker supports Containerfile

Reply:
Fixed; the test also created a docker file.